### DISCUSSION TITLE
INA219 increasing Volt resolution (or INA226, INA3221)

### DISCUSSION THREAD

**Discussion by NiklausS** [arendst/Tasmota#17350] [Q&A]
Hi,
I'm using esp and INA219 sensors for quite some time. Lately i wanted to move one solution to tasmota and i found that it somehow only displays the voltage without decimals. According to the spec it offers this:

> ADC basic resolution 12 12 bits
> Shunt voltage, 1 LSB step size 10 10 μV
> Bus voltage, 1 LSB step size 4 4 mV - this is the highside (0-26 V) so eg: 25.496V would be nicer then just 26 

is there any chance to set the voltage resolution ? (and W )

Thanks for the firmware - it's a pleasure to use it.

**Comment by sfromis** [comment]
https://tasmota.github.io/docs/Commands/#power-monitoring shows a number of commands to set resolution for different values, including what you ask for.

**Reply by NiklausS** [reply]
Thank you very much - i had the impression upon some readings they would be only for the energy part.

**Reply by sfromis** [reply]
When you have V, A and W, it is also energy, even if that's not what you're really after. And it is not only when measuring mains.....
